src/components/generic/form.tsx

```diff
@@ -0,0 +1,331 @@
+/* eslint-disable */
```

Comment:
What's the reason of this disabled lint?

Reply:
The form component is not yet finished with functions and needs refactoring, that should be removed after refactoring is done.